### PR TITLE
Mark HTTP semantic conventions as stable

### LINKED ISSUE #82: Mark HTTP semantic conventions as stable in the YAML definitions
Our semantic convention tooling defines a property called `stability` for semantic conventions.
Once we finally stabilize our HTTP conventions, the YAML files should be updated to set this property to `stable`.

See https://github.com/open-telemetry/build-tools/blob/main/semantic-conventions/syntax.md#syntax for details.

### PR DESCRIPTION
**Pull request by lmolkova**
Fixes #82 

## Merge requirement checklist

* [x] [CONTRIBUTING.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CONTRIBUTING.md) guidelines followed.
* [x] [CHANGELOG.md](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md) updated for non-trivial changes.
* ~~schema-next.yaml~~

### CODE REVIEW

**Comment by lmolkova** [comment]
some limitations we have in the md generator (and can address separately):
- only the original definition (in the registry) has a badge, referenced attributes don't get it
- semconv cannot be marked as stable as a whole
- metrics don't have stability levels in yaml

**Comment by AlexanderWert** [comment]
@lmolkova WDYT about rendering `deprecated` labels as well? Though I see that deprecated attributes have it also in their brief / description: `Deprecated, use ... instead.`

**Comment by lmolkova** [comment]
> @lmolkova WDYT about rendering `deprecated` labels as well? Though I see that deprecated attributes have it also in their brief / description: `Deprecated, use ... instead.`

They seem to be rendered by default - https://github.com/open-telemetry/build-tools/blob/370e8f443612b312d9d30679f4cd513152fdab1b/semantic-conventions/src/opentelemetry/semconv/main.py#L210

We just never use them in markdown, so they don't appear. If we did, they'd appear as
image

File: model/registry/http.yaml [hunk 1]
```diff
@@ -5,13 +5,15 @@ groups:
     brief: 'This document defines semantic convention attributes in the HTTP namespace.'
     attributes:
       - id: request.body.size
+        stability: stable
```
Comment (trask) [inline model/registry/http.yaml:8]: (#460)
```suggestion
```

File: model/registry/http.yaml [hunk 2]
```diff
@@ -91,13 +96,15 @@ groups:
           or any other).
         examples: 3
       - id: response.body.size
+        stability: stable
```
Comment (trask) [inline model/registry/http.yaml:99]:
```suggestion
```

**Comment by trask** [comment]
I think this should be "ready to review" as soon as these are done:
* #459
* #467
* #469
* #472

**Comment by trask** [comment]
@lmolkova can you expand this PR to also update the stability of the http spans page and relevant metrics sections, so that it will be the one PR that we merge related to marking things stable? thx

File: CHANGELOG.md [hunk 3]
```diff
@@ -61,6 +61,8 @@ release.
   ([#267](https://github.com/open-telemetry/opentelemetry-specification/pull/267))
 - Add opt-in `system.memory.limit` metric.
   ([#409](https://github.com/open-telemetry/semantic-conventions/pull/409))
+- Mark attributes used by HTTP semantic conventions as stable.
```
Comment (trask) [inline CHANGELOG.md:64]:
```suggestion
- Mark HTTP semantic conventions as stable.
```

**Review by trask — APPROVED** [review]

**Review by jsuereth — APPROVED** [review]

**Review by AlexanderWert — APPROVED** [review]

**Comment by Oberon00** [comment]
I have been asked to review this PR, I can (only) confirm that as far as I know, there are no blockers left for stability so probably this is good to go. But I haven't been following the spec/semconv work closely enough recently to add a formal review here.

**Review by joaopgrassi — APPROVED** [review]

**Review by jack-berg — APPROVED** [review]

**Pull request merged**